dev/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Step 1: Build the user's application
+FROM ajymau/java-openliberty-odo:0.1 as compile
+
+# Make a well known place for shared library jars seperate from the rest of the defaultServer contents (to help with caching)
+RUN mkdir /configlibdir \
+   && mkdir /config \
+   &&  mkdir /shared
+
+
+# Copy the rest of the application source
+COPY ./src /project/user-app/src
+COPY ./pom.xml /project/user-app/pom.xml
+
+# Build (and run unit tests) 
+#  also liberty:create copies config from src->target
+#  also remove quick-start-security.xml since it's convenient for local dev mode but should not be in the production image.
+RUN cd /project/user-app && \
+    echo "QUICK START SECURITY IS NOT SECURE FOR PRODUCTION ENVIRONMENTS. IT IS BEING REMOVED" && \
+    rm -f src/main/liberty/config/configDropins/defaults/quick-start-security.xml && \
+    mvn liberty:create package
+
+# process any resources or shared libraries - if they are present in the dependencies block for this project (there may be none potentially)
+# test to see if each is present and move to a well known location for later processing in the next stage
+# 
+RUN cd /project/user-app/target/liberty/wlp/usr/servers && \
+    if [ -d ./defaultServer/lib ]; then mv ./defaultServer/lib /configlibdir; fi && \
+    if [ ! -d /configlibdir/lib ]; then mkdir /configlibdir/lib; fi && \
+    mv -f defaultServer/* /config/ && \
+    if [ -d ../shared ]; then mv ../shared/* /shared/; fi
+
+# Step 2: Package Open Liberty image
+FROM openliberty/open-liberty:kernel-java8-openj9-ubi
```

Comment:
Should match our devfile version: OL 20.0.0.6

Reply:
actually I think Andy's changing to 20.0.0.9

Reply:
Updated to 20.0.0.9